src/arch/s390.go

```diff
@@ -0,0 +1,7 @@
+//go:build s390x
+// +build s390x
+
+package arch
+
+const Arch = ArchS390
```

Comment:
can you use dtclient.ArchS390 here aswell?

Reply:
Ammended this change:

I actually needed to do the inverse: take the Power architectures and fix their references to the package local (relative to the `arch` pkg) `ArchPPC` and `ArchPPCLE` vars.